src/index.js

```diff
@@ -503,7 +503,21 @@ const Sortable = React.createClass({
   }
 });
 
+const SortableContainer = ({
+  className, style, onMouseDown, onTouchStart, children,
+}) => (
+  <div
```

Comment:
By using `SortableContainer`, we might get different DOM structure since there will be another wrapping `<div>`, it may cause some css selectors not working properly. Maybe just `React.cloneElement(children, { someProps })`?